src/app/shared/pipes/order-view.pipe.ts

```diff
@@ -0,0 +1,90 @@
+import { formatDate } from '@angular/common';
+import { Pipe, PipeTransform } from '@angular/core';
+import { Order } from '../../features/trips/models/order.model';
+import { OrderForView } from '../../pages/orders-page/order/order.component';
+
+@Pipe({
+  name: 'transformOrder',
+})
+export class TransformOrderPipe implements PipeTransform {
+  // хелперы для преобразования seatId в typeCarriage и numberCarriage
+  private getTypeCarriage(seatId: number): string {
+    // Пример вычислений, заменить на реальные
```

Comment:
I think it's better to use English for comments

Reply:
I forgot to delete this pip, it's not in use.